wikimedia/wikipedia-ios
# Tap search tab button a second time to change focus to search bar

**Phabricator:** https://phabricator.wikimedia.org/T257708

### Notes
* This follows up from a community PR that didn't respond to follow up requests.

### Test Steps
1. Run branch.
2. Tap search bar button
3. Tap it a second time, search bar should become first responder (non-jargon: search bar focused and keyboard should appear). Type a few characters into the search bar.
4. Tap it another time, nothing should happen.
5. Press return, so cursor is no longer in search bar
6. Tap search tab bar button again, and it should again make the search bar the focus.
